File: Carrefour.Web/wwwroot/Scripts/Models/Employee.js
class Employee extends User
{
    constructor(json)
    {
        super(json);

        if (json)
        {
            this.Code = json.code;
            this.Category = json.category;
            this.Married = json.married;
            this.Transport = json.transport;
            this.Social = json.social;
            this.Turno = json.turno;
        }

        else
        {
            this.Code = 0;
            this.Category = "";
            this.Married = "";
            this.Transport = "";
            this.Social = 0;
            this.Turno = 0;
        }
    }
}

File: Carrefour.lib/Models/Employee.cs
namespace Carrefour.lib.Models
{
    public class Employee : User
    {
        public int Code { get; set; }
        public string Category { get; set; }
        public bool Married { get; set; }
        public bool Transport { get; set; }
        public int Social { get; set; }
        public int Turno { get; set; }
    }
}
